products/dev_alternate_products.md

```diff
@@ -0,0 +1,95 @@
+Details particular to an IntelliJ Platform-based IDE is described on the individual product pages in Part VIII.
+
+* bullet list
+{:toc}
+
+## Introduction 
+Plugins can extend or add functionality to IDEs based on the IntelliJ Platform. 
+This extensibility applies to a variety of JetBrains' IDEs, such as Android Studio, WebStorm, PhpStorm, and PyCharm.
+Plugins can be written in Kotlin or Java, or a mix of both, and are created using IntelliJ IDEA.
+Once completed, plugins can be packaged and distributed at [plugins.jetbrains.com](https://plugins.jetbrains.com).
+
+## Configuring Plugin Projects Targeting Alternate IntelliJ Platform-Based IDEs
+To create a new Gradle plugin project, follow the tutorial on the [Getting Started with Gradle](/tutorials/build_system/prerequisites.md) page.
+The tutorial produces a skeleton Gradle project suitable to use as a starting point.
+Some minor modifications are needed, as described below, and on the individual product pages in Part VIII.
+
+Understanding the relationship between build numbers is critical when developing a plugin to target an IntelliJ Platform-based IDE:
+* _targetIDE_ is the (version-specific) IntelliJ Platform-based IDE in which the plugin is intended to run, such as PyCharm.
+  The path in Gradle plugin attribute [`runIde.ideaDirectory`](/tutorials/build_system/gradle_guide.md#running-against-alternate-versions-and-types-of-intellij-platform-based-ides) is set to the installed location of _targetIDE_.
+* _baseIntelliJPlatformVersion_ is the (version-specific) IntelliJ Platform used in the build of the _targetIDE_.
+  The IntelliJ Platform is defined by a specific build of the IntelliJ IDEA Community Edition.
+  The Gradle plugin attribute [`intellij.version`](/tutorials/build_system/gradle_guide.md#intellij-platform-configuration) is set to be _baseIntelliJPlatformVersion_.
+
+For API compatibility, the IntelliJ Platform version used in the _targetIDE_ dictates the _baseIntelliJPlatformVersion_ used for developing a plugin. 
+
+### Matching Versions of the IntelliJ Platform with the Target IDE Version
+The _baseIntelliJPlatformVersion_ used in the _targetIDE_ may not be readily apparent, depending on the product.
+See the individual product pages in Part VIII for exceptions.
+
+To find the version of the IntelliJ Platform used to build the _targetIDE_, use the _About_ dialog screen for the _targetIDE_.
+Next to **Build #** is the BRANCH.BUILD.FIX version of the IntelliJ Platform used to build the _targetIDE_.
+In the example shown below, the (BRANCH.BUILD.FIX) version of the IntelliJ Platform is `192.6817.19`.
+The product version is 2019.2.3.
+
+![Example PyCharm](img/pycharm_build.png){:width="600px"}
+
+If the product version isn't clear on the _About_ screen, consult the individual product pages in Part VIII.
+
+Using [Toolbox](https://www.jetbrains.com/toolbox-app/) is an easy way to find build numbers for recent versions.
```

Comment:
we also publish all information readily on the website https://www.jetbrains.com/idea/download/other.html

Reply:
That is excellent! I will incorporate.